docs/commands/ft.aggregate.md

```diff
@@ -6,7 +6,7 @@ syntax: |
     [LOAD count field [field ...]] 
     [TIMEOUT timeout] 
     [ GROUPBY nargs property [property ...] [ REDUCE function nargs arg [arg ...] [AS name] [ REDUCE function nargs arg [arg ...] [AS name] ...]] ...]] 
-    [ SORTBY nargs [ property ASC | DESC [ property ASC | DESC ...]] [MAX num]] 
+    [ SORTBY nargs [ property ASC | DESC [ property ASC | DESC ...]] [MAX num] [WITHCOUNT] 

```

Comment:
@adrianoamaral `WITHCOUNT` and `WITHOUTCOUNT` can also be used without sorting
The location of these arguments should be related to the command itself.
Related #3651

Reply:
In this case, we won't mentioning WITHTOUTCOUNT